SeaPublicWebsite/Models/EnergyEfficiency/HeatingControlsViewModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using GovUkDesignSystem.Attributes.ValidationAttributes;
+using GovUkDesignSystem.ModelBinders;
+using Microsoft.AspNetCore.Mvc;
+using SeaPublicWebsite.BusinessLogic.Models.Enums;
+using SeaPublicWebsite.Resources;
+
+namespace SeaPublicWebsite.Models.EnergyEfficiency;
+
+public class HeatingControlsViewModel : QuestionFlowViewModel
+{
+    public string Reference { get; set; }
+
+    [ModelBinder(typeof(GovUkCheckboxEnumSetBinder<HeatingControls>))]
+    [GovUkValidateCustom(CustomValidationPropertyName = nameof(HeatingControlsIsValid),
+        ErrorMessageResourceType = typeof(ErrorMessages),
+        ErrorMessageResourceName = nameof(ErrorMessages.HeatingControlsRequired))]
+    public List<HeatingControls> HeatingControls { get; set; } = [];
+
+    public bool HeatingControlsIsValid => !(HeatingControls.Count == 0 ||
+                                            ((HeatingControls.Contains(BusinessLogic.Models.Enums.HeatingControls
+                                                  .DoNotKnow) ||
+                                              HeatingControls.Contains(BusinessLogic.Models.Enums.HeatingControls
+                                                  .None)) && HeatingControls.Count > 1));
```

Comment:
feels like this could be simplified somewhat. feels most intuitive to set up three conditions that make it valid, ie
- there is >1 option selected
- AND if do not know is selected, no more than 1 is selected
- AND if none is selected, no more than 1 is selected
may also help to split into 3 smaller booleans (`HeatingControlsIsValid` is a function, so feel free to define a function body and define some temporary variables)